src/styles/dialog-editor-boxes.scss

```diff
@@ -0,0 +1,141 @@
+.dialog-editor-container {
+  background-color: #fff;
+  padding: 20px;
+  margin-bottom: 10px;
+}
+
+.draggable-box {
+  background-color: $color-white-94;
+  border-radius: 1px;
+  cursor: pointer;
+  margin-right: 16px;
+  margin-top: 10px;
+  height: 100%;
+  width: 100%;
+  position: relative;
+
+  &.selected {
+    border: 3px solid $color-blue-300;
+  }
+
+  &:hover {
+    border: 1px solid $color-gray-1;
+    box-shadow: 0 2px 6px $color-black-20;
+
+    .header-fa {
+      float: right;
+      font-size: 22px;
+    }
+
+    &.selected {
+      border: 3px solid $color-blue-300;
+    }
+
+    .header-pf {
+      float: right;
+      font-size: 16px;
+      padding-top: 3px;
+      padding-right: 8px;
+    }
+
+    .show-on-hover {
+      display: initial !important;
+    }
+
+    &::before {
+      background-image: linear-gradient(to bottom, $color-lochmara-approx 60%, $color-white 0%);
+      background-position: left;
+      background-repeat: repeat-y;
+      background-size: 2px 5px;
+      border: 4px solid $color-lochmara-approx;
+      bottom: 4px;
+      content: "";
+      left: 4px;
+      position: absolute;
+      top: 3px;
+      width: 10px;
+    }
+  }
+
+  &:first-child {
+    margin-top: 0px;
+  }
+}
+
+.draggable-field {
+  background-color: $color-white-94;
+  border-radius: 1px;
+  cursor: pointer;
+  margin-right: 16px;
+  margin-top: 10px;
+  width: 100%;
+  padding-top: 9px;
+  padding-left: 20px;
+  padding-right: 8px;
+  padding-bottom: 2px;
+  position: relative;
+  z-index: 600;
+
+  &.selected {
+    border: 3px solid $color-blue-300;
+  }
+
+  &:hover {
+    border: 1px solid $color-gray-1;
+    box-shadow: 0 2px 6px $color-black-20;
+
+    &.selected {
+      border: 3px solid $color-blue-300;
+    }
+
+    button {
+      display: inherit !important;
+      position: relative;
+      top: 4px;
+    }
+
+    &::before {
+      background-image: linear-gradient(to bottom, $color-lochmara-approx 60%, $color-white 0%);
+      background-position: left;
+      background-repeat: repeat-y;
+      background-size: 2px 5px;
+      border: 4px solid $color-lochmara-approx;
+      bottom: 4px;
+      content: "";
+      left: 4px;
+      position: absolute;
+      top: 3px;
+      width: 10px;
+    }
+  }
+
+  &:first-child {
+    margin-top: 0px;
+  }
+
+  img {
+    height: 100%;
+    margin-right: 8px;
+    object-fit: contain;
+  }
+}
+
+.form-group {
```

Comment:
Don't we want to namespace everything in the CSS, I'm pretty sure that the `.form-group` can be also conflicting. Maybe not now, but what if someone just adds something here without thinking. cc @karelhala @himdel

Reply:
We're in a component (custom element) that can be used as a namespace.

Reply:
Definitely need to namespace everything. No way can we redefine `form-group` or similar without breaking everything in manageiq :) 

I think in the long run, we should remove everything from `src/styles/` in favor of per-component styles - which webpack would namespace by component automagically. But for now, something like `.dialog-editor .form-group` *might* suffice.

Reply:
Ok, removed ```.form-group```.  It wasn't being used and was an old artifact.  -thanks

Reply:
Reviewed the new .scss files added in this PR; everything appears to be namespaced or at least have unique parent class names.